README.md

```diff
@@ -1,42 +1,143 @@
-# The Process
+# Table of Contents
 
-This repository contains the source code for docker images that are used in [citus testing](https://github.com/citusdata/citus/blob/master/.circleci/config.yml).  
+* [Introduction](#Introduction)
+* [1. Images](#1-images)
+  * [1.1 How circleci uses our images in testing](#11-How-circleci-uses-our-images-in-testing)
+* [2. How to update images](#2-How-to-update-images)
+  * [2.1 How to update a postgres minor version in images](#2.1-How-to-update-postgres-minor-version-in-images)
+  * [2.2 How to add a major postgres major](#2.2-How-to-add-a-major-postgres-major)
+* [3. How to publish a change](#3-How-to-publish-a-change)
+
+## Introduction
+
+This repository contains the source code for docker images that are used in [citus testing](https://github.com/citusdata/citus/blob/master/.circleci/config.yml). The images are pushed to [docker hub](https://hub.docker.com/u/citus). There is no hooking logic between this repository and the docker hub account. It is used purely for the storage of our images source code.
```

Comment:
Can we make an issue to track a task to automate the image creation from a CI job? Running upgrades on developer machines have proven to be troublesome, as we had to create a new docker hub account due to access restrictions.

Reply:
#33.